src/main/java/seedu/duke/project/Project.java

```diff
@@ -53,8 +58,21 @@ public int getNumberTasks() {
      */
     @Override
     public String toString() {
-        
+
         return projectName;
-        
+
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return tasks;
+    }
+
+    public String selectTask(int taskId) {
+        try {
+            Task selectedTask = tasks.get(taskId);
+            return "Selected Task: " + selectedTask.toString();
+        } catch (NumberFormatException | IndexOutOfBoundsException | NullPointerException e) {
+            return "Task ID does not exist.";
+        }
```

Comment:
I feel that selectTask is just retrieving task using project.getTask() followed by an output message. There's no need for this function as output is handled in Command class